cmd/antrea-agent/agent.go

```diff
@@ -680,6 +680,18 @@ func run(o *Options) error {
 		return err
 	}
 
+	var secondaryNetworkController *secondarynetwork.Controller
+	if features.DefaultFeatureGate.Enabled(features.SecondaryNetwork) {
```

Comment:
How about moving this block to line 629 to be closer to CNIServer init? Please also add a comment saying secondary network Controller should be created before CNIServer.Run() to make sure no Pod CNI updates will be missed.

Reply:
done